src/UserRepository.js

```diff
@@ -20,6 +21,33 @@ class UserRepository {
     }, 0) / this.allSleepData.length;
     return Math.round(avgQuality * 10) / 10;
   }
+
+  getAvgStairs(date) {
+    return Math.round(this.allActivityData.reduce((acc,activity) => {
+      if (activity.date === date) {
+        acc += activity.flightsOfStairs;
+      }
+      return acc;
+    }, 0) * 10 / this.allActivityData.filter(activity => activity.date === date).length) / 10;
+  }
+
+  getAvgSteps(date) {
+    return Math.round(this.allActivityData.reduce((acc, activity) => {
+      if (activity.date === date) {
+        acc += activity.numSteps;
+      }
+      return acc;
+    }, 0) / this.allActivityData.filter(activity => activity.date === date).length)
+  }
+
+  getAvgMinutes(date) {
```

Comment:
`getAvgStairs()`, `getAvgSteps()`, and `getAvgMinutes()` all share similar code. Is there a way we could DRY up this code?